.env.sample

```diff
@@ -1,4 +1,5 @@
-ENVIRONEMT=
+FLASK_ENV=
```

Comment:
Why do we need this env var?

Reply:
For reloading of server, can do without it but it's not hurt to add